spec/unit/lib/wifi_user/use_cases/sponsor_users_spec.rb

```diff
@@ -146,8 +146,11 @@ def a_plural_sponsor_confirmation_request
     it 'Sends a sponsorship failed email to the sponsor' do
```

Comment:
I know this is pre-existing, but nit: casing

```suggestion
    it 'sends a sponsorship failed email to the sponsor' do
```

Reply:
```suggestion
    it 'sends a sponsorship failed email to the sponsor' do
```